HPOBenchExperimentUtils/tabular_ranking.py

```diff
@@ -0,0 +1,330 @@
+"""
+Script to generate the average ranking plots for the chosen optimizer families
+"""
+
+import pickle
+import logging
+import argparse
+from pathlib import Path
+from typing import Union, List
+
+from HPOBenchExperimentUtils.utils.plotting_utils import plot_dc, color_per_opt, unify_layout, \
+    benchmark_dc, benchmark_families, linestyle_per_opt
+from HPOBenchExperimentUtils import _log as _main_log
+from HPOBenchExperimentUtils.utils.validation_utils import load_json_files, load_trajectories_as_df
+from HPOBenchExperimentUtils.utils.runner_utils import get_optimizer_setting, get_benchmark_settings
+
+from hpobench.benchmarks.ml import TabularBenchmark
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+_main_log.setLevel(logging.DEBUG)
+_log = logging.getLogger(__name__)
+
+
+paper_tasks = [
+    10101, 53, 146818, 146821, 9952, 146822, 31, 3917, 168912, 3, 167119, 12, 146212, 168911,
+    9981, 167120, 14965, 146606, 7592, 9977
+]
+ntasks_done = dict(
+    svm=29,
+    lr=29,
+    rf=28,
+    xgb=22,
+    nn=8
+)
+
+
+def read_trajectories(benchmark: str, input_dir: Path, train: bool=True,
```

Comment:
also this seems like an exact copy of code in a different file. Could you at least indicate where you changed code?